### PR TITLE
Transcription errors and editorial changes

### PR DESCRIPTION
**Pull request by jennfrye**
Mainly these are transcription errors. The few editorial changes I made were a result of correcting those errors.

In the case of changing punctuation and capitalization (especially in my final commit), I expect that many of these are not errors so much as discrepancies, as it seems the Project Gutenberg transcription was based on a different set of source scans than the one SE is using. I didn't do a complete page-by-page comparison; the corrections I did make are those that I identified as possible errors when I was reading through the book, and then I searched the rest of the text for all related errors to make sure the changes are as uniform as possible (for example, I capitalized all cases of "General" that referred to the character, checked the capitalization following each short exclamation, etc.). 

All non-editorial changes I made are consistent with the source scans. 

Additional commits were made to the repository after I initially cloned it—I hope I rebased correctly so that this pull request can be merged. I remember this was a problem for me when I was working on "Villette."

### CODE REVIEW

File: src/epub/text/chapter-24.xhtml [hunk 1]
```diff
@@ -22,7 +22,7 @@
 			<p>“I did not expect to be able to return sooner, when I went away; but three hours ago I had the pleasure of finding nothing to detain me. You look pale. I am afraid I alarmed you by running so fast up those stairs. Perhaps you did not know⁠—you were not aware of their leading from the offices in common use?”</p>
 			<p>“No, I was not. You have had a very fine day for your ride.”</p>
 			<p>“Very; and does Eleanor leave you to find your way into all the rooms in the house by yourself?”</p>
-			<p>“Oh! No; she showed me over the greatest part on Saturday⁠—and we were coming here to these rooms⁠—but only”⁠—dropping her voice⁠—“your father was with us.”</p>
```
Comment (acabal) [inline src/epub/text/chapter-24.xhtml:25]: Closing `</p>` broken here

File: src/epub/text/chapter-2.xhtml [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
...ty, genius, accomplishment, nor manner. The air of a gentlewoman, a great deal of quiet, inactive good temper, and a trifling turn of mind were all that could account for her being the choice of a sensible, intelligent man like <abbr>Mr.</abbr> Allen. In one respect she was admirably fitted to introduce a young lady into public, being as fond of going everywhere and seeing everything herself as any young lady could be. Dress was her passion. She had a most harmless delight in being fine; and our heroine’s entrée into life could not take place till after three or four days had been spent in learning what was mostly worn, and her chaperone was provided with a dress of the newest fashion. Catherine too made some purchases herself, and when all these matters were arranged, the important evening came which was to usher her into the Upper Rooms. Her hair was cut and dressed by the best hand, her clothes put on with care, and both <abbr>Mrs.</abbr> Allen and her maid declared she looked quite as she should do. With such encouragement, Catherine hoped at least to pass uncensured through the crowd. As for admiration, it was always very welcome when it came, but she did not depend on it.</p>
```
Comment (acabal) [inline src/epub/text/chapter-2.xhtml]: Let's leave `chaperone` in all instances as that's the modern spelling

File: src/epub/text/chapter-27.xhtml [hunk 3]
```diff
@@ -14,7 +14,7 @@
...
...uct amiss. Pray explain everything to his satisfaction; or, if he still harbours any doubt, a line from himself to me, or a call at Putney when next in town, might set all to rights. I have not been to the rooms this age, nor to the play, except going in last night with the Hodges, for a frolic, at half price: they teased me into it; and I was determined they should not say I shut myself up because Tilney was gone. We happened to sit by the Mitchells, and they pretended to be quite surprised to see me out. I knew their spite: at one time they could not be civil to me, but now they are all friendship; but I am not such a fool as to be taken in by them. You know I have a pretty good spirit of my own. Anne Mitchell had tried to put on a turban like mine, as I wore it the week before at the concert, but made wretched work of it⁠—it happened to become my odd face, I believe, at least Tilney told me so at the time, and said every eye was upon me; but he is the last man whose word I would take. I wear nothing but purple now: I know I look hideous in it, but no matter⁠—it is your dear brother’s favourite colour. Lose no time, my dearest, sweetest Catherine, in writing to him and to me,</p>
```
Comment (acabal) [inline src/epub/text/chapter-27.xhtml:17]: The Pump Room is a historic named place so we should modernize `Pump-room` to `Pump Room`

**Review by acabal — CHANGES_REQUESTED** [review]
Great work, thanks! A few notes to work on before I can merge. Fixing the broken tag, and undoing `chaperon`, can be done by amending those commits instead of creating new commits. For `Pump Room` you can do that in an editorial commit. Thanks!

**Comment by acabal** [comment]
If you amend the commits don't forget to force push to this PR to update it.

**Comment by jennfrye** [comment]
Changes made! Thank you for the tips to amend and force push, I would have had a very difficult time figuring that out.

**Comment by acabal** [comment]
Excellent, thanks!

**Pull request merged**